.scalafmt.conf

```diff
@@ -0,0 +1,13 @@
+version = 3.5.1
+runner.dialect = scala213
+maxColumn = 120
+docstrings.style = Asterisk
+assumeStandardLibraryStripMargin = true
+rewrite.rules = [RedundantBraces, RedundantParens, SortImports, SortModifiers]
+indentOperator.exemptScope = all
+align.tokens = [
+  {code = "%", owner = "Term.ApplyInfix"},
+  {code = "%%", owner = "Term.ApplyInfix"},
+  {code = ":=", owner = "Term.ApplyInfix"},
+  "="
```

Comment:
So no alignment for cases?

```
{code = "=>", owner = "Case"}
```

With everything else aligned, may be good to have this too?

Reply:
Can you provide an example of current Scala code and what you expect the formatted to be? As far as I can see `case` is formatted but maybe I am misunderstanding?

Reply:
I thought I was cases that weren't aligned.

Reply:
Have commented on a couple of examples where this was aligned before.